README.md

```diff
@@ -96,15 +96,15 @@ We want fields to be added to the original form in a specific order, intermixed
 
 ### Extending the form with direct properties
 
-Some properties are defined directly on the form, e.g. the generator. If we want to extend the form with such properties, we should define them on the `form:Extension`. The form-content service will take the `form:Extension` instance and treat it as a `form:Form` instance. It will examine the form instance it extends and add all of its predicates to itself.
+For our extension, we want to be able to specify the same direct properties as we do with a normal form. In the case of our extension, we should define them on the `form:Extension`, instead of on the `form:Form`. The form-content service will take the `form:Extension` instance and treat it as a `form:Form` instance. It will examine the form instance it extends and add all of its predicates to itself.
```

Comment:
```suggestion
In an extension, we should be able to specify the same direct properties that we use in a regular form. In that case, we should define them on the `form:Extension`, instead of on the `form:Form` using the same predicates as the ones used on the `form:Form`. The form-content service will take the `form:Extension` instance and transform it into a `form:Form` instance that contains the combination of the new properties and the properties that exist on the form that is being extended.
```